src/main/java/com/easyrest/components/UserRoleMenuPanel.java

```diff
@@ -0,0 +1,29 @@
+package com.easyrest.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import com.easyrest.components.Header;
+
+public class UserRoleMenuPanel {
+    protected WebDriver driver;
+
+    private final By userPanelButton = By.xpath("//ul[@role='menu']//child::a");
+    private final By logOutButton = By.xpath("//ul[@role='menu']//child::li");
+
+    Header header = new Header(driver);
```

Comment:
The header component is not a part of the menu panel, but the menu panel is a part of the header.